ext/src/Cassandra/Collection.c

```diff
@@ -42,18 +46,20 @@ php_cassandra_collection_find(cassandra_collection* collection, zval* object, lo
 {
   HashPointer ptr;
   zval**      current;
-  zval        compare;
   ulong       idx;
+  cassandra_type_collection* type;
+
+  type = (cassandra_type_collection*) zend_object_store_get_object(collection->type TSRMLS_CC);
 
-  if (!php_cassandra_validate_object(object, collection->type TSRMLS_CC))
+  if (!php_cassandra_validate_object(object, type->value_type TSRMLS_CC))
```

Comment:
Since the return is on the next line how do you feel about bracing or inlining the return?